src/plugins/intel_cpu/tests/functional/single_layer_tests/conversion.cpp

```diff
@@ -68,6 +68,42 @@ class ConvertCPULayerTest : public testing::WithParamInterface<convertLayerTestP
 
         function = makeNgraphFunction(ngPrc, params, conversion, "ConversionCPU");
     }
+
+    void generate_inputs(const std::vector<ngraph::Shape>& targetInputStaticShapes) override {
```

Comment:
@dmitry-gorokhov could you take a look?

Reply:
To convert f32 to bool, the output of CPU plugin, ceil(abs(float)), is a u8 value. It retain the format of u8. But the output of ngraph ref is literal bool, which is either 0 or 1. We set the input floating points to be in the range of [0, 1), so no extra precision converting between actual output and expected ouput will be needed from the test side.

Reply:
1. Could you please use [-1, 1] range?
2. Please add description why we need custom inputs initialization

Reply:
Good point! Fixed. Thanks Dmitry!